active-rfcs/0000-relative-url-scheme.md

```diff
@@ -0,0 +1,95 @@
+- Start Date: 2021-12-20
+- Reference Issues: [#46](https://github.com/withastro/rfcs/pull/46)
+- Implementation PR: <!-- leave this empty -->
+
+# Summary
+
+Astro should support source-relative URL resolution in the HTML block of Astro files.
+
+# Example
+
+To resolve the source-relative URL of an image, use a `local:` prefixed URL.
+
+```html
+<img src="local:kitten.avif" alt="Kitten" />
+```
+
+# Motivation
+
+In `.astro` files, it’s not always easy to know what a relative URL is relative to.
+
+```astro
+<img src="kitten.avif" alt="kitty" />
+```
+
+In this example, is `kitten.avif` relative to the output `.html` or the source `.astro`? Or, if this example is found within a component `.astro` file, is it relative to that component file?
+
+This ambiguity raises a need for:
+
+- An intuitive way to author URLs relative to a source file.
+- An intuitive way to author URLs within the HTML of an `.astro` file.
+
+# Detailed design
+
+A `local` scheme instructs Astro to resolve the URL relative to the source file.
```

Comment:
I would expect this section to get into the *how* Astro will resolve.

For example, we had talked about offline that these would be translate to dynamic imports. If that's the case it should be explained in this section.

Reply:
I’ve added those details to the section. Let me know if they address this well.